Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1

```diff
@@ -1901,52 +1901,56 @@ function Import-CrmSolutionAsync{
 		catch
 		{
 			throw LastCrmConnectorException($conn)
-		}    
+		}
+
         $pollingStart = Get-Date
         $isProcessing = $true
 		$secondsSpentPolling = 0
-        $pollingDelaySeconds = 5
+        $pollingDelaySeconds = 60
 		$transientFailureCount = 0; 
-        Write-Verbose "Import of file completed, waiting on completion of AsyncOperationId: $importId"
+        Write-Verbose "Solution import requested, waiting on completion..."
```

Comment:
I would suggest keeping the importId in the verbose messages to enable easier troubleshooting.

Reply:
I've removed all occurrences of $importId except the first one (line 1891), because I thought that this would be enough for debugging and there was no reason to repeatedly print it, but ok I'll put it back.